codegen/type_converter.go

```diff
@@ -71,14 +71,26 @@ type TypeConverter struct {
 	Helper        PackageNameResolver
 	uninitialized map[string]*fieldStruct
 	fieldCounter  int
+
+	requestTypeHelper RequestHelper
+}
+
+// RequestHelper is the helper struct for method generation
+type RequestHelper struct {
```

Comment:
Can we rename this to `ConvertOptions` and rename `Request` => `From` and `Response` => `To`